src/BowlingGame.js

```diff
@@ -0,0 +1,79 @@
+var BowlingGame = function() {
+	this.rolls = [];
+	this.throw = 0;
+  this.turn = 0;
+  this.frame = 0;
+  this.frameScore = [];
+  this.gameOver = false;
+  this.scoreSheet =[];
+};
+
+BowlingGame.prototype.roll = function(pins) {
```

Comment:
This function has too many statements. (14)<br>This function's cyclomatic complexity is too high. (5)